eZ/Publish/API/Repository/ContentService.php

```diff
@@ -427,6 +427,26 @@ public function loadTranslationInfos(ContentInfo $contentInfo, array $filter = a
     public function removeTranslation(ContentInfo $contentInfo, $languageCode);
 
     /**
+     * Delete specified Translation from a Content Draft.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified Translation
+     *         is the only one the Content Draft has or it is the main Translation of a Content Object.
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed
+     *         to edit the Content (in one of the locations of the given Content Object).
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if languageCode argument
+     *         is invalid for the given Draft.
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if specified Version was not found
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\VersionInfo $versionInfo Content Version Draft
+     * @param string $languageCode Language code of the Translation to be removed
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Content Content Draft w/o the specified Translation
+     *
+     * @since 6.12
+     */
+    public function deleteTranslation(VersionInfo $versionInfo, $languageCode);
```

Comment:
Maybe `removeTranslationFromDraft`?

Having both, `deleteTranslation` and `removeTranslation`, but operating on different data will be very confusing.

Reply:
> Maybe removeTranslationFromDraft?
> Having both methods, deleteTranslation and removeTranslation, but operating on different data will be very confusing.

Could be. I chose this name to align with `deleteContent` and `deleteVersion`.
`removeTranslation` refers to removing from entire content so this is nice distinction. It had been chosen due to Legacy reasons
// cc @andrerom